content/01-install/02a-install-wallet-windows.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Install Wallet on Windows
+metaTitle: Install Wallet on Windows
+---
+>Author's note to reviewers: I understand that the Mantis wallet is a full-node wallet, so it does need disk space to sync. What about mining? Please clarify.
```

Comment:
There does not seem to be a feature for enabling mining in the wallet. (I suppose you could edit some conf files if you are so inclined, but I don't really see the point.)